migrations/migrations/20190521213634-populate_users.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
If you want to populate the database, you need to use seeds instead of migrations.  http://docs.sequelizejs.com/manual/migrations.html, look for seeds section.